crates/wallet/src/lib.rs

```diff
@@ -329,32 +328,32 @@ pub async fn fetch_inputs_from_db_or_node(
     let total_amount_available =
         db::proof::compute_total_amount_of_available_proofs(db_conn, node_id)?;
 
-    if total_amount_available < target_amount {
+    if u64::from(total_amount_available) < target_amount {
```

Comment:
Maybe it is easier to change `target_amount` and all other amounts used to `Amount`.
Now that `compute_total_amount_of_available_proofs` and all other db functions are using the type `Amount`